wagtailio/settings/base.py

```diff
@@ -397,19 +392,44 @@
 }
 
 
-# Sentry
+# Sentry configuration.
+# See instructions on the intranet:
+# https://intranet.torchbox.com/delivering-projects/tech/starting-new-project/#sentry
+is_in_shell = len(sys.argv) > 1 and sys.argv[1] in ["shell", "shell_plus"]
 
-if "SENTRY_DSN" in env:
-    INSTALLED_APPS += ("raven.contrib.django.raven_compat",)
-    RAVEN_CONFIG = {"dsn": env["SENTRY_DSN"]}
-    try:
-        RAVEN_CONFIG["release"] = raven.fetch_git_sha(PROJECT_ROOT)  # noqa
-    except InvalidGitRepository:
+if "SENTRY_DSN" in env and not is_in_shell:
+
+    import sentry_sdk
+    from sentry_sdk.integrations.django import DjangoIntegration
+    from sentry_sdk.utils import get_default_release
+
+    sentry_kwargs = {
+        "dsn": env["SENTRY_DSN"],
+        "integrations": [DjangoIntegration()],
+    }
+
+    # There's a chooser to toggle between environments at the top right corner on sentry.io
+    # Values are typically 'staging' or 'production' but can be set to anything else if needed.
+    # dokku config:set gosh SENTRY_ENVIRONMENT=staging
```

Comment:
`gosh` is Torchbox client app I think? Probably better to remove this line as wagtail.io is not hosted using dokku.